adapters/repos/db/aggregator/vector_search.go

```diff
@@ -90,11 +90,7 @@ func (a *Aggregator) buildAllowList(ctx context.Context) (helpers.AllowList, err
 
 	if a.params.Filters != nil {
 		s := a.getSchema.GetSchemaSkipAuth()
-		allow, err = inverted.NewSearcher(a.logger, a.store, s, nil,
-			a.classSearcher, a.deletedDocIDs, a.stopwords, a.shardVersion,
-			a.isFallbackToSearchable, a.tenant, a.nestedCrossRefLimit).
-			DocIDs(ctx, a.params.Filters, additional.Properties{},
-				a.params.ClassName)
+		allow, err = inverted.NewSearcher(a.logger, a.store, s, nil, a.propertyIds, a.classSearcher, a.deletedDocIDs, a.stopwords, a.shardVersion, a.isFallbackToSearchable, a.tenant, a.nestedCrossRefLimit).DocIDs(ctx, a.params.Filters, additional.Properties{}, a.params.ClassName)
```

Comment:
Can we revert this file to it's original state? as there are no changes only the formatting, which makes it hard to read the method we somebody uses side by side tabs when developing code.